plugins/inputs/opentelemetry_listener/opentelemetry.go

```diff
@@ -0,0 +1,92 @@
+package opentelemetry_listener
+
+import (
+	"net"
+	"sync"
+
+	"github.com/influxdata/influxdb-observability/otel2influx"
+	otlpcollectorlogs "github.com/influxdata/influxdb-observability/otlp/collector/logs/v1"
+	otlpcollectormetrics "github.com/influxdata/influxdb-observability/otlp/collector/metrics/v1"
+	otlpcollectortrace "github.com/influxdata/influxdb-observability/otlp/collector/trace/v1"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"google.golang.org/grpc"
+)
+
+type OpenTelemetryListener struct {
+	ServiceAddress string `toml:"service_address"`
+
+	grpcServer    *grpc.Server
+	grpcServerErr error
+	listener      net.Listener
+
+	wg sync.WaitGroup
+
+	Log telegraf.Logger `toml:"-"`
```

Comment:
Can you please move it next to `ServiceAddress` to have all exported members in one spot!?

Reply:
Sure. TBH, I'm not clear on why this is exported. Just following the pattern from other plugins.